### PR TITLE
Consolidate environment variables in one file

### PR DESCRIPTION
**Pull request by therealplato**

### CODE REVIEW

File: deployment/config.go [hunk 1]
```diff
@@ -4,18 +4,12 @@ import (
...
 
-var (
-	// EnvServiceHost is the Kubernetes host
-	EnvServiceHost = os.Getenv("KUBERNETES_SERVICE_HOST")
-	// EnvServicePort is the Kubernetes port
-	EnvServicePort = os.Getenv("KUBERNETES_PORT_443_TCP_PORT")
+	"k8s.io/kubernetes/pkg/client/restclient"
 )
 
 func init() {
```
Comment (poopoothegorilla) [inline deployment/config.go]: porque?

File: env/env.go [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+func LoadEnvs() {
+	SlackToken = loadAndWarn("SLACK_VERIFICATION_TOKEN")
+	ServerHost = loadAndWarn("HOST")
+	K8sServiceHost = loadAndWarn("KUBERNETES_SERVICE_HOST")
+	K8sServicePort = loadAndWarn("KUBERNETES_PORT_443_TCP_PORT")
+	K8sNamespace = loadAndWarn("KUBERNETES_NAMESPACE")
+	EtcdHost = loadAndWarn("ETCD_HOST")
+}
+
+func loadAndWarn(key string) (val string) {
```
Comment (poopoothegorilla) [inline env/env.go]: Do you think we need to explicitly state `AndWarn`? it doesn't look like we will need a regular `load` function... what do you think?

Comment (poopoothegorilla) [inline env/env.go]: not really a big deal at all though

**Comment by poopoothegorilla** [comment]
+1

File: env/env.go (hunk 2 above)
Comment (therealplato) [inline env/env.go]: I named it what it does!
We may want a loadOrFail function for any mandatory variables

**Pull request merged**